data/json/monstergroups/wilderness.json

```diff
@@ -3,22 +3,23 @@
     "type": "monstergroup",
     "name": "GROUP_CAVE",
     "default": "mon_null",
-    "//": "Current count is 925.  Note that 'freq' units are tenth of a percent, with default filling in the gap.",
+    "//": "Current count is 975.  Note that 'freq' units are tenth of a percent, with default filling in the gap.",
     "is_animal": true,
     "monsters": [
-      { "monster": "mon_bat", "freq": 600, "cost_multiplier": 5, "pack_size": [ 6, 32 ] },
+      { "monster": "mon_bat", "freq": 550, "cost_multiplier": 5, "pack_size": [ 6, 32 ] },
       { "monster": "mon_bear", "freq": 100, "cost_multiplier": 10, "pack_size": [ 1, 3 ] },
       { "monster": "mon_cougar", "freq": 100, "cost_multiplier": 20, "pack_size": [ 1, 2 ] },
+      { "monster": "mon_mole_cricket", "freq": 60, "cost_multiplier": 3 },
       { "monster": "mon_mutant_experimental", "freq": 25, "cost_multiplier": 3 },
       { "monster": "mon_nakedmolerat_giant", "freq": 100, "cost_multiplier": 3 },
-      { "monster": "mon_mole_cricket", "freq": 60, "cost_multiplier": 3 }
+      { "monster": "mon_stag_beetle_larva", "freq": 40, "cost_multiplier": 5 }
     ]
   },
   {
     "type": "monstergroup",
     "name": "GROUP_FOREST",
     "default": "mon_null",
-    "//": "Current SPRING first DAY count is 513.  Note that 'freq' units are tenth of a percent, with default filling in the gap.",
+    "//": "Current SPRING first DAY count is 514.  Note that 'freq' units are tenth of a percent, with default filling in the gap.",
```

Comment:
Double-check this please; should go up from 513 to 519 (by 6, not 1), unless the earlier number was miscounted.

Reply:
I believe that because this count is for spring day 1, I should only count the group that spawns on day 1.  If that's incorrect, I'll make sure to fix it.